lib/shopify_cli/constants.rb

```diff
@@ -58,6 +58,18 @@ module EnvironmentVariables
       MONORAIL_REAL_EVENTS = "MONORAIL_REAL_EVENTS"
     end
 
+    module SupportedVersions
+      module Ruby
+        FROM = "2.6.6"
+        TO = "3.0.2"
+      end
+
+      module Node
+        FROM = "12.0.0"
+        TO = "17.0.0"
+      end
```

Comment:
I think we should mention this in the [documentation](https://shopify.dev/apps/tools/cli/installation). For example, we say `Ruby 2.7 or higher` as a requirement, but if they choose the latest version (3.1), then the CLI will show a warning. What do you think @shainaraskas?

Reply:
Well, Ruby 3.1 was released around 6 weeks ago already 😅 , but I see your point. Another option could be to add a comment in this very piece of code to remember updating the docs. As you prefer!